app/controllers/episodes_controller.rb

```diff
@@ -12,6 +12,12 @@ def edit; end
 
   def new; end
 
+  def download
+    @episode = Episode.find(params[:episode_id])
+    VideoDownloadJob.perform_later(@episode)
+    redirect_to '/'
```

Comment:
Why do we need to redirect to root on download?

Reply:
Most likely this path will be used as api, and I added this redirect for the test